routes/api/articles.js

```diff
@@ -58,4 +58,16 @@ router.post('/:slug/like', verifyJwt(), ArticleController.likeArticle);
 
 router.post('/:slug/dislike', verifyJwt(), ArticleController.dislikeArticle);
 
+router.get(
```

Comment:
Since you didn't use try and catch in your controller you can try to use ```./helpers/asyncHandler.js``` on this callback function. For more info and how to use it ask @oesukam

Reply:
This is someone else work!